package.json

```diff
@@ -14,8 +14,12 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.20.0",
+    "cassandra-driver": "^4.6.0",
+    "csv-writer": "^1.6.0",
     "express": "^4.17.1",
+    "express-cassandra": "^2.5.0",
     "faker": "^5.1.0",
+    "fs": "0.0.1-security",
```

Comment:
did you do npm install fs? If so, you don't have to change this but I think fs is one of those dependencies that node knows about so you don't have to npm install it. Just like "path". Unless there's a reason why you needed to. Just thought i'd let you know, otherwise, Great job Stu!!! You code is looking very clean, I'm sure Sam would approve lol.

Reply:
Good catch, I usually forget that. I can remove the dependency for the next PR.